src/model/documents/category.ts

```diff
@@ -0,0 +1,8 @@
+import { BaseModel } from '../base-model';
+import { CategoryType } from '../enums/category-type';
+
+export interface Category extends BaseModel {
+  imagePath: string;
```

Comment:
It could be a custom type for URL, cause images will be served via Firebase Storage CDN in the future.

Reply:
Also as it is in Poster, be can add an optional property for category background color as a fallback, if user will not provide image.